Do not use -isystem for Boost library
If installed, MacPorts Boost headers are found
--- client/dump/CMakeLists.txt.orig	2019-08-14 07:25:47.000000000 -0700
+++ client/dump/CMakeLists.txt	2019-09-17 19:53:52.000000000 -0700
@@ -28,7 +28,7 @@
     -DBOOST_SYSTEM_NO_DEPRECATED)
 ENDIF()
 
-INCLUDE_DIRECTORIES(SYSTEM ${BOOST_PATCHES_DIR} ${BOOST_INCLUDE_DIR})
+INCLUDE_DIRECTORIES(${BOOST_PATCHES_DIR} ${BOOST_INCLUDE_DIR})
 
 SET(MYSQLPUMP_LIB_SOURCES
   abstract_chain_element.cc
--- mysys/CMakeLists.txt.orig	2020-07-21 00:00:00.000000000 +0000
+++ mysys/CMakeLists.txt	2020-07-24 19:20:00.000000000 -0500
@@ -30,7 +30,7 @@
   TARGET_COMPILE_OPTIONS(mytime_objlib PRIVATE $<$<CONFIG:RelWithDebInfo>:-xO1>)
 ENDIF()
 
-INCLUDE_DIRECTORIES(SYSTEM ${BOOST_PATCHES_DIR} ${BOOST_INCLUDE_DIR})
+INCLUDE_DIRECTORIES(${BOOST_PATCHES_DIR} ${BOOST_INCLUDE_DIR})
 
 SET(MYSYS_SOURCES
   array.cc
--- plugin/keyring_udf/CMakeLists.txt.orig	2020-07-21 00:00:00.000000000 +0000
+++ plugin/keyring_udf/CMakeLists.txt	2020-07-24 19:03:00.000000000 -0500
@@ -20,7 +20,7 @@
 # along with this program; if not, write to the Free Software
 # Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301  USA
 
-INCLUDE_DIRECTORIES(SYSTEM ${BOOST_PATCHES_DIR} ${BOOST_INCLUDE_DIR})
+INCLUDE_DIRECTORIES(${BOOST_PATCHES_DIR} ${BOOST_INCLUDE_DIR})
 
 MYSQL_ADD_PLUGIN(keyring_udf
   keyring_udf.cc
--- sql/CMakeLists.txt.orig	2020-07-21 00:00:00.000000000 +0000
+++ sql/CMakeLists.txt	2020-07-24 18:54:00.000000000 -0500
@@ -20,7 +20,7 @@
 # along with this program; if not, write to the Free Software
 # Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301  USA
 
-INCLUDE_DIRECTORIES(SYSTEM ${BOOST_PATCHES_DIR} ${BOOST_INCLUDE_DIR})
+INCLUDE_DIRECTORIES(${BOOST_PATCHES_DIR} ${BOOST_INCLUDE_DIR})
 
 MY_INCLUDE_SYSTEM_DIRECTORIES(ICU)
 
--- unittest/gunit/CMakeLists.txt.orig	2020-07-21 00:00:00.000000000 +0000
+++ unittest/gunit/CMakeLists.txt	2020-07-24 19:01:00.000000000 -0500
@@ -25,7 +25,7 @@
 ENDIF()
 
 # Some tests require Boost.
-INCLUDE_DIRECTORIES(SYSTEM ${BOOST_PATCHES_DIR} ${BOOST_INCLUDE_DIR})
+INCLUDE_DIRECTORIES(${BOOST_PATCHES_DIR} ${BOOST_INCLUDE_DIR})
 
 INCLUDE_DIRECTORIES(SYSTEM ${GMOCK_INCLUDE_DIRS})
 
